### ISSUE TITLE
Add bifold to Bifoldable

### ISSUE THREAD

**Issue by LukaJCB** [typelevel/cats#2947]
Something like this:

```scala
def bifold[F[_, _]: Bifoldable, A: Monoid, B: Monoid](fab: F[A, B]): (A, B) =
  bifoldMap(fab)(a => (a, Monoid[B].empty))(b => (Monoid[A].empty, b))
```

**Comment by Twizty** [comment]
Hi, is it still wanted? I'd like to implement that. I guess the previous attempt is abandoned.

**Comment by travisbrown** [comment]
This is closed in #3088.
